packages/uni_ui/lib/profile/profile_card.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/material.dart';
+import 'package:phosphor_flutter/phosphor_flutter.dart';
+import 'package:uni_ui/generic_card.dart';
+
+class ProfileCard extends StatelessWidget {
+  const ProfileCard({
+    super.key,
+    required this.label,
+    required this.content,
+    this.tooltip,
+  });
+
+  final String label;
+  final String content;
+  final String? tooltip;
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      clipBehavior: Clip.none,
+      children: [
+        GenericCard(
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                label,
+                overflow: TextOverflow.ellipsis,
+                style: Theme.of(context).textTheme.titleSmall!,
+              ),
+              Text(
+                content,
+                overflow: TextOverflow.ellipsis,
+              ),
+            ],
+          ),
+        ),
+        if (tooltip != null)
+          Positioned(
+            bottom: -4,
+            right: 2,
+            child: Tooltip(
```

Comment:
Tooltip is for the card. Should we include an optional tooltip in the generic card?